src/functionalTests/utils/mockAwalaMiddleware.ts

```diff
@@ -18,8 +17,6 @@ const EXPECTATIONS: Expectation[] = [
     httpRequest: {
       method: 'POST',
       path: '/',
-      // eslint-disable-next-line @typescript-eslint/naming-convention
-      headers: { 'Content-Type': AwalaContentType.PDA },
```

Comment:
You have to remove the whole object containing the `httpRequest` and `httpResponse`: We're no longer expecting the PDA.

So `EXPECTATIONS` must have exactly one `Expectation`.